packaging/skupper-router.spec

```diff
@@ -65,6 +65,8 @@ BuildRequires: cmake
 # skupper-router requirements
 BuildRequires: python3-devel
 BuildRequires: python3-setuptools
+BuildRequires: python3-setuptools_scm
```

Comment:
this one is probably not necessary, I don't have time right now to play with removing it

Reply:
it is not necessary, removing it